Week3/homework/step2-7.js

```diff
@@ -20,4 +20,4 @@ f2(y);
 
 console.log(y);
 
-// Add your explanation as a comment here
+// in both examples, the actual value of x changes. Because both of the functions use the arguments which are passed as reference, not as value.
```

Comment:
As you will have seen, the value of `x` doesn't change in the first case. Therefore your explanation is incorrect.

Reply:
Hi @remarcmij ,
It is my bad. I did it very quicklky to study for the exam. I realized that my answer is incorrect. The new explanation could be seen in my new commit. Thanks for your valuable feedback. :pray: